app/jobs/descriptive_metadata_import_job.rb

```diff
@@ -45,4 +46,19 @@ def open_version(cocina_object, description)
   rescue RuntimeError => e
     Failure([e.message])
   end
+
+  def validate_and_save(cocina_object, description)
+    result = CocinaValidator.validate_and_save(cocina_object, description:)
+    return Success(cocina_object) if result.success?
+
+    Failure(["validate_and_save failed for #{cocina_object.externalIdentifier}"])
+  end
+
```

Comment:
`CocinaValidator` does not return the object, which we need to pass onto `close_version` ... this new method wraps that action and allows us to pass the cocina object in the success result